tests/conftest.py

```diff
@@ -33,6 +33,9 @@ def event_loop():
 def cleanup_children():
     yield
 
+    with RSAPrivateKey._process_wide_key_lock:
+        RSAPrivateKey._process_wide_key = None
```

Comment:
Didn't see this change in the description and right now it seems to have no effect, do we need this?

Reply:
This has no effect, but it will if you run existing tests in different order.

TL;DR here is how it breaks things:
- you create **something** that instantiates RSAPrivateKey.process_wide
- and then all subsequent tests use the same key
- crucially, if you create two DHT instances with RSA validator **AFTER** instantiating private key, everything breaks because they both inherit your key

Future PRs introduce tests that instantiate validators before the sensitive tests and break everything in bizarre ways.